### PR TITLE
create class continue option wait for schema creation

### PR DESCRIPTION
**Pull request by sadakchap**

### CODE REVIEW

File: src/dashboard/Data/Browser/Browser.react.js [hunk 1]
```diff
@@ -1088,7 +1089,8 @@ class Browser extends DashboardView {
 
     const classes = await Parse.Schema.all();
     const schema = classes.find( c => c.className === this.props.params.className);
-
+    console.log(`classes ${classes}`);
```
Comment (nataliaconde) [inline src/dashboard/Data/Browser/Browser.react.js:1092]: Could you please remove this console.log()?

File: src/dashboard/Data/Browser/Browser.react.js [hunk 2]
```diff
@@ -1088,7 +1089,8 @@ class Browser extends DashboardView {
...
+    console.log(`currentClassName`, this.props.params.className)
```
Comment (nataliaconde) [inline src/dashboard/Data/Browser/Browser.react.js:1093]: Could you please remove this console.log()?

File: src/dashboard/Data/Browser/Browser.react.js [hunk 3]
```diff
@@ -1156,6 +1158,7 @@ class Browser extends DashboardView {
       }
       this.setState({ data: data, filters, lastMax: MAX_ROWS_FETCHED , filteredCounts: filteredCounts, fetchedErr: null });
     } catch(err) {
+      console.log(err);
```
Comment (nataliaconde) [inline src/dashboard/Data/Browser/Browser.react.js:1161]: Could you please remove this console.log()?

File: src/dashboard/Data/Browser/Browser.react.js (hunk 1 above)
Comment (sadakchap) [reply src/dashboard/Data/Browser/Browser.react.js:1092]: Sure thing.

**Review by nataliaconde — APPROVED** [review]

**Pull request merged**